app/editor/public/links.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Not sure if this is the best approach

Reply:
This won't work because of how images are built and deployed.  I'll dig into how we're getting this value in other places.  I can't recall if we have something similar for the web app.

Reply:
Thanks!

Reply:
Ah I forget more than I remember regrettably.  I've a solution for this and your prior PR that used a constant.  You can fix both together.

The API has an Admin page for Settings.  You can configure a Key=Value pair for anything you need.  Then you can access the value from the Redux store in the lookups I believe.

![image](https://github.com/bcgov/tno/assets/3180256/60bc818f-21c4-4067-9141-05832e345d29)

Reply:
Perfect, thank you!